Source/bmalloc/libpas/src/libpas/pas_log.c

```diff
@@ -31,7 +31,13 @@
 
 #include <errno.h>
 #include "pas_snprintf.h"
+#if PAS_OS(WINDOWS)
+#include <io.h>
+#define write _write
```

Comment:
Instead of using macro, we should define a static function and inside it we should switch `write` or `_write`.